test/uk/gov/hmrc/emcstfe/services/SubmitChangeDestinationServiceSpec.scala

```diff
@@ -88,5 +89,20 @@ class SubmitChangeDestinationServiceSpec extends TestBaseSpec
         }
       }
     }
+
+    "formatErrorForLogging" must {
+      "return the errorType if the errorType is not 12 or 13" in new Test {
+        val validationError: RIMValidationError = RIMValidationError(errorCategory = Some("business"), errorType = Some(1), errorReason = Some("some error"), errorLocation = Some("some location"))
+        service.formatErrorForLogging(validationError) shouldBe "Some(1)"
+      }
+
+      "return Some(errorType) (errorReason: reason) if the errorType is 12 or 13" in new Test {
+        val validationError12: RIMValidationError = RIMValidationError(errorCategory = Some("business"), errorType = Some(12), errorReason = Some("some error"), errorLocation = Some("some location"))
+        val validationError13: RIMValidationError = RIMValidationError(errorCategory = Some("business"), errorType = Some(13), errorReason = Some("some error"), errorLocation = Some("some location"))
+
+        service.formatErrorForLogging(validationError12) shouldBe "Some(12) (errorReason: Some(some error))"
+        service.formatErrorForLogging(validationError13) shouldBe "Some(13) (errorReason: Some(some error))"
```

Comment:
Out of interest (and I know you didn't do this as part of this PR) but - why does it have `Some()` and `Some()`. I thought initially it was Options flowing through, but then I saw the code actually adds `"Some()"` around it? Which doesn't really seem needed?

Reply:
I guess for consistency when searching through the logs. Not sure honestly.